index.yaml

```diff
@@ -34,7 +34,6 @@ distributions:
       nightly-fastrtps-dynamic: eloquent/ci-nightly-fastrtps-dynamic.yaml
       nightly-navigation2: eloquent/ci-nightly-navigation2.yaml
       nightly-navigation2-prerequisites: eloquent/ci-nightly-navigation2-prerequisites.yaml
-      nightly-opensplice: eloquent/ci-nightly-opensplice.yaml
```

Comment:
This is removing the job for Eloquent, not Foxy.

I missed this, but noticed when I tried to redeploy the config.

Reply:
Fixed in #66.